tools/data_export/export_remote.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="export_remote" name="Export datasets" version="0.1.0">
+    <description>to remote files source</description>
+    <command><![CDATA[
+        python '$__tool_directory__/export_remote.py'
+            --file-sources '$file_sources'
+            --directory-uri '$d_uri'
+            #if $export_type.export_type_selector == "datasets_auto"
+                --infiles
+                #for $filepath in $export_type.infiles:
+                    '$filepath'
+                #end for
+                --names
+                #for $name in $export_type.infiles:
+                    #if $name.ext == 'vcf_bgzip':
+                        #set file_ext = 'vcf.bz'
+                    #else:
+                        #set file_ext = $name.ext
+                    #end if
+                    '${name.element_identifier}.${file_ext}'
```

Comment:
@jmchilton Add tool wrapper support for doing this natively. If the extension is already included don't duplicate it.

Reply:
Something like https://github.com/galaxyproject/galaxy/pull/3220 ? I was going to pick this back up, but let me know if you have other plans.